sys/suit/handlers_common.c

```diff
@@ -48,18 +43,26 @@ static int _component_handler(suit_manifest_t *manifest, int key,
      * ]
      * */
     nanocbor_value_t arr;
-    if (nanocbor_enter_array(&_it, &arr) < 0) {
+    if (nanocbor_enter_array(it, &arr) < 0) {
```

Comment:
Can you explain the change?

Reply:
Yup, the component structure used to be bstring wrapped in a previous version. This handler first had to unpack and parse the cbor inside the bstring object to get the number of components in the manifest. This bstring wrapping got removed in the latest version, so no more need to parse the bstring first, the cbor reference to the component list can be passed directly to the `enter_array()` here.